netlify.toml

```diff
@@ -0,0 +1,28 @@
+# BUILD Settings:
+# ##############################################################################
+[build]
+  publish = "public"
+
+[dev]
+  publish = "public"
+  port = 64946
+  autolaunch = true
```

Comment:
I can remove this if we don't want to support `netlify dev` I think we stopped using it for Giving and SAA site?

Reply:
It comes in handy every once in a while when trying to troubleshoot something in a build or function but we very rarely need to use it now.

Reply:
I see - I'll probably remove it in the next PR then. We can always add it back if we need it later. Trying to be good about only adding things that we use 😄

Reply:
PR ok to merge?